utils/logging.py

```diff
@@ -0,0 +1,9 @@
+from loguru import logger
+
+
+class Logger:
+    @staticmethod
+    def set_logger():
+        logger.remove()
+        return logger.add("debug.log", format="{time:DD:MM:YYYY:h:m:s} {level} {message}", level="DEBUG",
```

Comment:
would be nice to have different files with different levels debug, error for example

Reply:
There is no need for it. 
You said several times that we need only low level logging.
Debug gonna be in file, Error gonna be last string in file.